CRM/ManualDirectDebit/DAO/RecurrMandateRef.php

```diff
@@ -0,0 +1,199 @@
+<?php
+
+require_once 'CRM/Core/DAO.php';
+require_once 'CRM/Utils/Type.php';
+
+class CRM_ManualDirectDebit_DAO_RecurrMandateRef extends CRM_Core_DAO {
+
+  /**
+   * Static instance to hold the table name.
+   *
+   * @var string
+   */
+  static $_tableName = 'dd_contribution_recurr_mandate_ref';
+
+  /**
+   * Static entity name.
+   *
+   * @var string
+   */
+  static $entityName = 'RecurrMandateRef';
+
+  /**
+   * Should CiviCRM log any modifications to this table in the civicrm_log
+   * table.
+   *
+   * @var boolean
+   */
+  static $_log = TRUE;
+
+  /**
+   * Unique ID
+   *
+   * @var int unsigned
+   */
+  public $id;
+
+  /**
+   * Recurr id
+   *
+   * @var int
+   */
+  public $recurr_id;
+
+  /**
+   * Mandate id
+   *
+   * @var int
+   */
+  public $mandate_id;
+
+  /**
+   * Returns all the column names of this table
+   *
+   * @return array
+   */
+  static function &fields() {
+    if (!isset(Civi::$statics[__CLASS__]['fields'])) {
+      Civi::$statics[__CLASS__]['fields'] = [
+        'id' => [
+          'name' => 'id',
+          'type' => CRM_Utils_Type::T_INT,
+          'title' => ts('id'),
+          'description' => 'id',
+          'required' => TRUE,
+          'import' => TRUE,
+          'where' => self::getTableName() . '.id',
+          'headerPattern' => '',
+          'dataPattern' => '',
+          'export' => TRUE,
+          'table_name' => self::getTableName(),
+          'entity' => self::getEntityName(),
+          'bao' => 'CRM_ManualDirectDebit_BAO_RecurrMandateRef',
+        ],
+        'recurr_id' => [
+          'name' => 'recurr_id',
+          'type' => CRM_Utils_Type::T_INT,
+          'title' => ts('Recurr id'),
```

Comment:
better to be more Descriptive in the title here, so instead of 'Recurr id
 use 'Reccuring Contribution ID'